libs/utils/energy_model.py

```diff
@@ -1056,7 +1082,8 @@ def simplified_em_present_in_cpusysfs(target, directory=None):
             if em_present:
                 _log.info('Attempting to load EM using {}'.format(load.__name__))
                 em = load(target, **args)
-                data_to_json(em)
+                with open("em_from_target.yml", "w") as f:
```

Comment:
This should be moved in a separate method like save_yaml(yaml_path) so it is only done explicitly with the user deciding of the filename